service/history/service.go

```diff
@@ -122,7 +122,7 @@ func (s *Service) Start() {
 // Stop stops the service
 func (s *Service) Stop() {
 	s.logger.Info("ShutdownHandler: Evicting self from membership ring")
-	_ = s.membershipMonitor.EvictSelf()
+	_ = s.membershipMonitor.EvictSelf(time.Time{})
```

Comment:
optional nit: I think `EvictSelfNow()` (and `EvictSelfAt(...)` in other places) might be a bit more readable